### ISSUE TITLE
Not working for me...

### ISSUE THREAD

**Issue by dflorey** [andreasbm/lit-translate#12]
I followed the steps outlined in your example (using exactly the same files etc.), but when using

${translate("header.title")}

in my template all I get is

(part) => { partCache.set(part, cb); updatePart(part, cb); }

instead of the localized message.
Ideas?

**Comment by runem** [comment]
This usually happens if you have multiple different versions of `lit-html` installed in your **node_modules** at the same time. 

What's the output if you run `npm list lit-html` to find all versions of lit-html in your node_modules?

**Comment by dflorey** [comment]
Ah, good point... I guess it's time to go through upgrade hell :-)
Thanks for your speedy reply!

+-- @formsey/fields-material@0.1.11
| `-- @material/mwc-checkbox@0.3.6
|   +-- @material/mwc-base@0.3.6
|   | `-- lit-html@1.0.0  deduped
|   `-- @material/mwc-ripple@0.3.6
|     `-- lit-html@1.0.0  deduped
+-- lit-element@2.0.1
| `-- lit-html@1.0.0
`-- lit-translate@1.1.21
  `-- lit-html@1.1.2

**Comment by runem** [comment]
No problems, I'm happy to help :-)

**Comment by andreasbm** [comment]
Thanks for helping resolve this issue @runem! 👍

**Comment by nicolasr75** [comment]
@runem @andreasbm 
I get the same error now and I think it began after I included Googles material web compontents.

My output for npm list lit-html is this:

+-- @material/mwc-button@0.15.0
| +-- @material/mwc-ripple@0.15.0
| | `-- lit-html@1.2.1  deduped
| `-- lit-html@1.2.1
+-- @material/mwc-icon-button@0.12.0
| `-- @material/mwc-ripple@0.12.0
|   `-- lit-html@1.2.1  deduped
+-- @material/mwc-textfield@0.15.0
| +-- @material/mwc-floating-label@0.15.0
| | `-- lit-html@1.2.1  deduped
| +-- @material/mwc-line-ripple@0.15.0
| | `-- lit-html@1.2.1  deduped
| `-- lit-html@1.2.1  deduped
+-- lit-element@2.3.1
| `-- lit-html@1.2.1  deduped
`-- lit-translate@1.2.1
  `-- lit-html@1.2.1  deduped

So, there are no different versions. Any idea what could be wrong?

**Comment by dflorey** [comment]
It may happen if you have the same lit-html version loaded from 2 different locations.
I managed to get rid of the error by forcing webpack to load both from the same location like this:

```
    resolve: {
        alias: {
          "lit-html": path.resolve(__dirname, 'node_modules/lit-html')
        ]
...
```

**Comment by nicolasr75** [comment]
Hmm, I don't use webpack. I just use ES6 'bare module specifiers' and run it through 'polymer build' to replace these with the actual paths. I would have to check whether there is anything like 'resolve' for Polymer build.

**Comment by nicolasr75** [comment]
I deleted all node modules and started from scratch. First Polymer stuff and lit-element, checked that it works and finally material web components. It still works. I remember that I initially had it the other way around. The material web components did install lit-element automatically but something was missing. I then installed lit-element explicitly which fixed that error but obviously caused the error described in this issue. So the problem is fixed for me :-)
